fix(core): make demo layouts responsive

aighita opened the pull request:
Closes https://github.com/kestra-io/kestra/issues/12738.

<!-- Thanks for submitting a Pull Request to Kestra. To help us review your contribution, please follow the guidelines below:

- Make sure that your commits follow the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) specification e.g. `feat(ui): add a new navigation menu item` or `fix(core): fix a bug in the core model` or `docs: update the README.md`. This will help us automatically generate the changelog.
- The title should briefly summarize the proposed changes.
- Provide a short overview of the change and the value it adds.
- Share a flow example to help the reviewer understand and QA the change.
- Use "closes" to automatically close an issue. For example, `closes #1234` will close issue #1234. -->

### What changes are being made and why?

UI Secrets page was missing mobile responsive styles.

<img width="1903" height="1011" alt="image" src="https://github.com/user-attachments/assets/60c18d7b-9416-4bf1-aed1-59089269c064" />

<!-- Please include a brief summary of the changes included in this PR e.g. closes #1234. -->

---

### How the changes have been QAed?

<img width="1897" height="1016" alt="image" src="https://github.com/user-attachments/assets/3017111e-77e6-4306-a284-0a67566ec7cf" />

https://github.com/user-attachments/assets/d197f50c-f773-4699-b523-1f14d23aee23

<!-- Include example code that shows how this PR has been QAed. The code should present a complete yet easily reproducible flow.

```yaml
# Your example flow code here
```

Note that this is not a replacement for unit tests but rather a way to demonstrate how the changes work in a real-life scenario, as the end-user would experience them.

Remove this section if this change applies to all flows or to the documentation only. -->

---

### Setup Instructions

Followed Kestra Dev Container README: https://github.com/kestra-io/kestra/tree/develop/.devcontainer with Local mode

<!--If there are any setup requirements like API keys or trial accounts, kindly include brief bullet-points-description outlining the setup process below.

- [External System Documentation](URL)
- Steps to set up the necessary resources

If there are no setup requirements, you can remove this section.

Thank you for your contribution. ❤️ Don't forget to give us a star! ⭐  -->

aighita committed:
fix(ui): Secrets page - mobile responsive.

MilosPaunovic commented:
Just a small note for the future, if you could use **GitHub Keywords** when opening PRs to connect it to the proper issue (to automate the entire process a bit more), it would be super helpful to us. You can read a bit more about it [here](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests#linking-a-pull-request-to-an-issue) (I've added closing keyword for this particular issue).

aighita commented:
Hi @MilosPaunovic! Sure, I will. Although this does not actually close #12738 . This was a bug I saw while exploring the project, on the Secrets page while no secrets are added.

For the EE pages issue I asked to be assigned I was planning to open a separate one, but I can commit to this same branch if that'd be better.

MilosPaunovic commented:
Yes please, let's handle it all in a single PR, this one.

MilosPaunovic committed:
Merge branch 'develop' into fix/secrets-page-mobile

aighita committed:
fix(ui): Updated EE demo/Layout.vue to be fully responsive across all device sizes.

aighita commented:
Fixed Audit Logs page that used Layout.vue component. Closes #12738 
Updated `ui/src/components/demo/Layout.vue` to be fully responsive across all device sizes.

<img width="430" height="925" alt="image" src="https://github.com/user-attachments/assets/577633e7-d775-4fdc-8669-785a6fce62eb" />

Layout.vue component is used in all the other EE pages inside the demo/ folder too.

<img width="297" height="330" alt="image" src="https://github.com/user-attachments/assets/ce344af4-3d67-4073-a4a8-1a5d293760d5" />

So this also fixed other pages/components like Tests EE:

<img width="393" height="844" alt="image" src="https://github.com/user-attachments/assets/2c894ea6-39b1-4b74-b65c-0184f67182ff" />

<img width="389" height="841" alt="image" src="https://github.com/user-attachments/assets/15bb4c45-6609-43bc-9118-de1bf7e2c608" />


& the components seen in the 2nd image.

Piyush-r-bhaskar committed:
fix: QOL improvement

Piyush-r-bhaskar reviewed (approved):
LGTM 

Thanks much for the contribution @aighita 🚀
